lib/isMobilePhone.js

```diff
@@ -70,7 +70,7 @@ var phones = {
   'tr-TR': /^(\+?90|0)?5\d{9}$/,
   'uk-UA': /^(\+?38|8)?0\d{9}$/,
   'vi-VN': /^(\+?84|0)?((1(2([0-9])|6([2-9])|88|99))|(9((?!5)[0-9])))([0-9]{7})$/,
-  'zh-CN': /^(\+?0?86\-?)?1[3456789]\d{9}$/,
+  'zh-CN': /^((\+|00)86)?1([358][0-9]|4[579]|66|7[0135678]|9[89])[0-9]{8}$/,
```

Comment:
Did you just change to a more specific validator here, more like a subset of the previous one?

Reply:
This rule is more stringent, thank you.

Reply:
Strict verification is not necessary. China's phone numbers are increasing very fast, and new numbers will be added every once in a while. The use of strict verification will be difficult to maintain.